app.go

```diff
@@ -421,6 +421,11 @@ type Static struct {
 	// Optional. Default value 0.
 	MaxAge int `json:"max_age"`
 
+	// SetHeaders defines a function that sets custom headers on response.
+	//
+	// Optional. Default: nil
+	SetHeaders func(c *Ctx)
```

Comment:
actually you can do more in this method than just setting the headers

you could change everything

so i was thinking more of a wording like in the proxy paket

https://github.com/gofiber/fiber/blob/00ebb216d4d2b3b52211b4c8989e2ee0ada83e1d/middleware/proxy/config.go#L31-L34

@hi019 @efectn what do you think ?

Reply:
Agree